master/contrib/github_buildbot.py

```diff
@@ -104,10 +105,10 @@ def render_POST(self, request):
             request.setResponseCode(OK)
             return json.dumps({"result": "pong"})
 
-        # Reject non-push, non-ping events
-        if event_type != "push":
+        # Reject events not involving code changes
+        if event_type not in ("push", "pull_request"):
```

Comment:
This kind of event handling does not really allow to easily extend the list of events to handle.  Would you consider implementing something like [this](https://github.com/buildbot/buildbot/blob/master/master/buildbot/www/hooks/github.py#L51)?